app/models/comment.rb

```diff
@@ -1,3 +1,12 @@
 class Comment < ApplicationRecord
   belongs_to :user, class_name: 'User'
```

Comment:
- [ ] A comment belongs to the author, kindly consider using author instead of user e.g `belongs_to :author, class_name: 'User'
`

Reply:
Dear Ismailco,

Changing my user model to author breaks the code block especially when inserting the post using rails c